util/SitRepEntry.cpp

```diff
@@ -180,7 +180,7 @@ SitRepEntry CreateCombatDamagedObjectSitRep(int object_id, int combat_system_id,
         sitrep.AddVariable(VarText::DESIGN_ID_TAG,     boost::lexical_cast<std::string>(ship->DesignID()));
 
     } else if (TemporaryPtr<const Planet> planet = boost::dynamic_pointer_cast<const Planet>(obj)) {
-        if (planet->Unowned())
+        if (planet->Owner()!=empire_id)
```

Comment:
I think this is fine as it was. If the test passes, it creates an unonwed system bombarded message. Why should it do this for planets owned by another empire?

Also, put spaces around the !=